london_cafes/london_cafes/templates/index.html

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
Do you need to set the title block here too? You have defined that page title must be a title block in the base